### DISCUSSION TITLE
Can add Dark theme?

### DISCUSSION THREAD

**Discussion by xNKSQx** [OpenTabletDriver/OpenTabletDriver#2230] [Ideas]
I want Dark theme on opentabletdriver cause i burning eyes from white light

**Comment by jamesbt365** [comment]
Use linux and macos where they have native theming, or beg Microsoft to add native theming to Wpf, or wait until 0.7 where we might replace it to WinUI3 which has native themeing.

**Reply by jamesbt365** [reply]
Resolved by #2772
